tasks/sign-base64-blob/sign-base64-blob.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: sign-base64-blob
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create an internalrequest to sign a base64 encoded blob.
+  params:
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
+    - name: request
+      type: string
+      description: Signing pipeline name to handle this request
+      default: "blob-signing-pipeline"
+    - name: blob
+      type: string
+      description: The base64 encoded blob to be signed.
+    - name: requester
+      type: string
+      description: Name of the user that requested the signing, for auditing purposes
+    - name: requestTimeout
+      type: string
+      default: "180"
+      description: InternalRequest timeout
+    - name: binariesPath
+      type: string
+      description: The directory inside the workspace where the binaries are stored
+      default: binaries
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: sign-base64-blob
+      image:
+        quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c
+      script: |
+        #!/usr/bin/env sh
+        set -ex
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        request=$(jq -r '.sign.request // "$(params.request)"' ${DATA_FILE})
+        default_pipeline_image="quay.io/redhat-isv/operator-pipelines-images:released"
+        pipeline_image=$(jq -r --arg default_pipeline_image ${default_pipeline_image} \
+            '.sign.pipelineImage // $default_pipeline_image' ${DATA_FILE})
+        config_map_name=$(jq -r '.sign.configMapName // "signing-config-map"' ${DATA_FILE})
+
+        echo "Creating InternalRequest to sign image:"
```

Comment:
We're not signing image here, right?

Also, excuse my ignorance, but how does this actually work? The same isv-provided pipeline that normally signs images also excepts `blob` instead of `reference` and then it will sign whatever string you give it?

Reply:
Awesome, thanks!

The text should still be modified. Probably to `"Creating InternalRequest to sign blob:"`

Reply:
That's 100% true! Changed! Thanks!